csharp/src/Ice/ConnectionI.cs

```diff
@@ -2515,6 +2533,13 @@ private void sendResponse(OutgoingResponse response, bool isTwoWay, byte compres
                         sendMessage(new OutgoingMessage(response.outputStream, compress > 0, adopt: true));
                     }
 
+                    if (_state == StateActive && _maxDispatches > 0 && _dispatchCount == _maxDispatches)
+                    {
+                        // Resume reads if the connection is active and the dispatch count is (about to be) bellow the
+                        // max dispatch count.
```

Comment:
Do you want to rename the field _maxDispatchCount?

Reply:
I don't have a preference. I'm fine with `_maxDispatchCount` if you prefer it.

Reply:
I prefer _maxDispatches like the property name. We should use the same terminology in the comments.